data/data/aws/main.tf

```diff
@@ -19,11 +19,33 @@ module "bootstrap" {
   target_group_arns        = "${module.vpc.aws_lb_target_group_arns}"
   target_group_arns_length = "${module.vpc.aws_lb_target_group_arns_length}"
   vpc_id                   = "${module.vpc.vpc_id}"
-  vpc_security_group_ids   = "${list(module.vpc.master_sg_id)}"
+  vpc_security_group_ids   = "${list(module.vpc.master_sg_id, aws_security_group.public_ssh.id)}"
```

Comment:
Can you drop [this][1], now that your new group makes it redundant?

[1]: https://github.com/openshift/installer/blob/ee26337a50fc67db480292a07e0a6b0fc8ea17fa/data/data/aws/bootstrap/main.tf#L148-L156

Reply:
In fact, do we want to drop the bootstrap security group altogether and have the bootstrap module add [the journald gateway rule][1] to one of the passed in groups (and have it removed during bootstrap teardown)?  That's more efficient, but somewhat breaks down the isolation between this module and the bootstrap module.

[1]: https://github.com/openshift/installer/blob/ee26337a50fc67db480292a07e0a6b0fc8ea17fa/data/data/aws/bootstrap/main.tf#L158-L166